### PR TITLE
first run at rewrite

### PR DESCRIPTION
**Pull request by cooktheryan**

### CODE REVIEW

File: labs/7.md [hunk 1]
```diff
@@ -1,90 +1,127 @@
...
+Now that the Mongo cluster has been configured, it is time to deploy the *pacman* application.
+There are many different types of federated objects but they are somewhat similar to those
+non-federated objects. For more information about federated objects see the following  [examples](https://github.com/kubernetes-sigs/kubefed/tree/master/example/sample1) and
+the [user guide](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md).
 
-* [cssl](https://pkg.cfssl.org/R1.2/cfssl_linux-amd64)
-* [cfssljson](https://pkg.cfssl.org/R1.2/cfssljson_linux-amd64)
+The pacman namespace must be created and then defined as a federated namespace.
+~~~sh
+oc new-project pacman
```
Comment (mvazquezc) [inline labs/7.md]: s/oc new-project pacman/oc create ns pacman/

File: labs/7.md [hunk 2]
```diff
@@ -1,90 +1,127 @@
...
+oc create -n pacman -f 04-pacman-federated-service-account.yaml
+# Create the pacman cluster role
+oc create -n pacman -f 05-pacman-federated-cluster-role.yaml
+# Create the clusterrolebinding
+oc create -n pacman -f 06-pacman-federated-cluster-role-binding.yaml
+# Create the deployment
+oc create -n pacman -f 07-pacman-federated-deployment-rs.yaml
+~~~
+
+## Deploying HAProxy
```
Comment (mvazquezc) [inline labs/7.md]: This could be potentially removed from the lab instructions, the HAProxy will be pre-deployed.

Comment (mvazquezc) [reply labs/7.md]: Or maybe moved to a different file where we can expand information related to the labs, wdyt?

File: labs/8.md [hunk 3]
```diff
@@ -1,153 +1,39 @@
-## Deploying MongoDB
-There are many different types of federated objects but they are somewhat similar to those
-non-federated objects. For more information about federated objects see the following  [examples](https://github.com/kubernetes-sigs/kubefed/tree/master/example/sample1) and
-the [user guide](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md).
+# Application Portability
+The strength of KubeFed is the ability to mange application workloads and move applications between clusters.
 
-
-The first step is to clone the demo code to your local machine:
+## Moving the Application
```
Comment (mvazquezc) [inline labs/8.md]: I'll add an example patching the placement so students can see the difference between modifying the replicas (deployment scaled to 0) and modifying the placement (deployment deleted)

File: labs/7.md (hunk 2 above)
Comment (cooktheryan) [reply labs/7.md]: I would say we move it once we get the agnosticd stuff in place and we can test against that

File: labs/8.md (hunk 3 above)
Comment (cooktheryan) [reply labs/8.md]: card created

File: labs/5.md [hunk 4]
```diff
@@ -5,13 +5,13 @@ clusters have already been deployed using of the deployment mechanisms defined a
...
-The first step is to decide which of the clusters will run the Federation Operator.
-Only one cluster runs the Kubefed controller manager.
+The first step is to decide which of the clusters will run the KubeFed Operator.
+Only one cluster runs the KubeFed controller manager.
 
 A new project of *federated-mongo* is on the command line in cluster1. Once the project is created the next step is to deploy the operator.
 
 ~~~sh
-oc new-project federated-mongo
+oc new ns federated-mongo
```
Comment (mvazquezc) [inline labs/5.md]: s/new/create/

File: labs/1.md [hunk 5]
```diff
@@ -1,21 +1,15 @@
...
 
-This demo is a simple deployment of [Federation Operator](https://operatorhub.io/operator/alpha/federation.v0.0.10) on two OpenShift
-clusters. A sample application is deployed to both clusters through the Federation controller.
-
-The files within this directory are used with the Federation operator to show
-MongoDB running on multiple OpenShift clusters. An accompanying video
-is [here](https://youtu.be/StNJupmZ5Mo). This demonstration uses 3 OpenShift 4 clusters. It is assumed that 3 OpenShift
-clusters have already been deployed using of the deployment mechanisms defined at
-https://cloud.openshift.com.
+This demo is a simple deployment of [Kubefed Operator](https://operatorhub.io/operator/kubefed-operator) on multiple OpenShift
```
Comment (scollier) [inline labs/1.md]: small nit, should be "KubeFed" everywhere, instead of Kubefed or kubefed.

File: labs/6.md [hunk 6]
```diff
@@ -1,72 +1,160 @@
...
 ~~~
-The next step is to federate the clusters using `kubefedctl`.
+
+Now that the yaml files contain the `pem` and routes it is time to deploy the objects.
 ~~~sh
-kubefedctl join east1 --host-cluster-context east1 --add-to-registry --v=2 --federation-namespace=federated-mongo
-kubefedctl join east2 --host-cluster-context east1 --add-to-registry --v=2 --federation-namespace=federated-mongo
-kubefedctl join west2 --host-cluster-context east1 --add-to-registry --v=2 --federation-namespace=federated-mongo
+# Create secret with admin/access credentials
+oc --context=east1 -n federated-mongo create -f 01-mongo-federated-secret.yaml
```
Comment (scollier) [inline labs/6.md]: will these be the actual contexts we use in the lab?  east1, west2, etc..?

Comment (scollier) [reply labs/6.md]: should we rename them "cluster1", and "cluster2" or something they may grok easier?

**Pull request merged**